src/ui/units/connections/components/custom-forms/Yadocs/components/YadocListItemView/UploadedYadocView.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import {I18n} from 'i18n';
+import {parseError} from 'ui/utils/errors/parse';
+
+import {ConverterErrorCode} from '../../../../../constants';
+import type {UploadedYadoc} from '../../../../../store';
+import {ListItem, ListItemAction} from '../../../components';
+// import type {HandleItemClick} from '../../types';
```

Comment:
Are these the forgotten comments here and below?

Reply:
Yep. I'll fix it